bun/whatsapp-with-vonage/src/main.ts

```diff
@@ -0,0 +1,67 @@
+import jwt from "jsonwebtoken";
+import { fetch } from "undici";
+
+export default async ({ req, res, log, error }) => {
+  if (req.method === "GET") {
+    const fileName = Bun.resolveSync("../static/index.html", import.meta.dir);
+    const html = await Bun.file(fileName).text();
+
+    return res.send(html, 200, {
+      "Content-Type": "text/html; charset=utf-8",
+    });
+  }
+
+  try {
+    const hasher = new Bun.CryptoHasher("sha256");
+    const hashedBuffer = hasher.update(JSON.stringify(req.body)).digest("hex");
+    const hashedValue = hashedBuffer.toString();
+
+    const authHeader = req.headers.authorization.split(" ")[1];
+    const decodedToken = jwt.verify(
+      authHeader,
+      Bun.env.VONAGE_SIGNATURE_SECRET,
+      {
+        algorithms: ["HS256"],
+      }
+    );
+    if (hashedValue != decodedToken["payload_hash"]) {
+      return res.json({
+        ok: false,
+        error: "Payload mismatched",
+      });
+    }
+  } catch (err) {
+    return res.json({
+      ok: false,
+      error: "can't verify",
+    });
+  }
```

Comment:
Please use the same flow as the existing node template. This is completely different